subcommittees/trainers/minutes/2017-01.txt

```diff
@@ -0,0 +1,103 @@
+Software and Data Carpentry Trainers Meeting 2017-01-10
+
+Please add yourself to the sign-in list below.
+	* Greg Wilson
+	* Jason Williams
+	* Jonah Duckles
+	* Lex Nederbragt
+	* Tim Dennis
+	* Karin Lagesen
+	* Sue McClatchy
+	* Mateusz Kuzak
+	* Anelda van der Walt
+	* Rayna Harris
+	* Tracy Teal
+	* Karen Cranston
+	* Maneesha Sane
+	* Erin Becker (popped on at end)
+
+Agenda
+	1. Succession
+	2. Publishing Instructor Training Materials
+	3. Recent Instructor Training Events
+	4. Upcoming Training Events
+	5. Reorganizing Trainer Training
+
+1. Succession
+
+	* The EDs and staff are discussing plans to reallocate DIT responsibilities
+		* Maneesha and Erin to take over some responsibilities, new co-hire from both Carpentrires being worked out
+		* Greg to continue being involed in instructor-training
```

Comment:
“involed” → “involved”.

I'm also not sure what “DIT” means a few lines up.  It may be worth de-acronyming that.